hack/setup-test-cluster.sh

```diff
@@ -18,11 +18,37 @@ do
     ${OC_TOOL} label $node node-role.kubernetes.io/worker-sctp=""
 done
 
+
+# Note: this is intended to be the only pool we apply all mcs to.
+# Additional mcs must be added to this poll in the selector
+cat <<EOF | ${OC_TOOL} apply -f -
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: test-pool
+  labels:
+    test-pool: ""
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+          key: machineconfiguration.openshift.io/role,
+          operator: In,
+          values: [worker-sctp, worker],
+        }
+  maxUnavailable: null
+  paused: false
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-sctp: ""
```

Comment:
maybe use here a more generic node selector if this is going to be for all the features?

Reply:
I preferred this way because teh sctp tests rely on that selector.
When we add new nodes, we can extend that selector (iirc the perf operator still relies on node labels to some extent).

Reply:
when the performance operator starts using this MCP, it will look a little odd to have to specify the worker-sctp node role in the performance profile config.

Reply:
Can't we have the nodeSelector matching both labels (rt & sctp) ?

Reply:
Ok, reworked it.
@davidvossel @SchSeba can you give a look?

Reply:
This works for me now thanks @fedepaol